fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -64,6 +64,29 @@
     </field>
   </xsl:template>
 
+
+  <!-- the following template creates an _ms field for committee members -->
+  <xsl:template match="mods:mods/mods:name[(mods:role/mods:roleTerm='Committee member') or 
+    (mods:role/mods:roleTerm='committee member')]" mode="utk_ir_MODS">
+
+    <xsl:for-each>
```

Comment:
Same comments as on PR #10 for TRAC-886; this stylsheet won't compile: `xsl:for-each` has a **mandatory** select attribute. 

This won't quite work the way the template is articulated. Take a look at the [`xsl:for-each` for mods:abstract(s)](https://github.com/utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config/blob/institutional_repository/fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt#L85-L92).